src/en/tut-lxd.md

```diff
@@ -0,0 +1,264 @@
+Title: Getting started with Juju and LXD
+
+# Getting started with Juju and LXD
+
+[LXD](lxd-upstream) provides an amazing experience using machine containers to be able to
+create the same models that are created with Juju on other public and private
+clouds. Using LXD as a method to test, verify, and replicate complex software
+deployments is a powerful tool that every Juju user needs. These instructions
+will get you up and running and deliver the best-possible experience with
+Juju. At the moment, that means using the latest release of Ubuntu with '[Long
+Term Support][long-term-support]' (LTS): [16.04 LTS
+                                          (Xenial)][Xenial-download].
+
+Our system will need the following:
+
+- [LXD][LXD-upstream]: a hypervisor for LXC, providing fast, secure containers.
+- [ZFS][ZFS-wiki]: a highly efficient and feature-rich filesystem and logical volume manager.
+
+
+## Install the software
+
+Begin by installing the required software. You'll need Juju and LXD on the
+machine. This means that using LXD is not supported on the breadth of Juju
+supported operating systems.
+
+For more options in installing Juju see the [install docs][install].
+
+
+```bash
+sudo snap install --classic juju
```

Comment:
we don't need to install Juju. This is a tutorial on setting up LXD for use with Juju. I think at some point we have to assume that people have actually installed Juju, either from 'getting started or the install page'. We do still need to install  lxd and zfsutils-linux though. 

Perhaps this could be solved simply by supplying a link to install the Juju client in the paragraph above?